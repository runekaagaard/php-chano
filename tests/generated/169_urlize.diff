001- <a href="http://google.com/" rel="nofollow">http://google.com/</a>
001+ Fatal error: Call to undefined method DtlIter::urlize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/169_urlize.php on line 4