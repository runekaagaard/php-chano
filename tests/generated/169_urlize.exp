<a href="http://google.com/" rel="nofollow">http://google.com/</a>