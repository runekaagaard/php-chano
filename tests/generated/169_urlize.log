
---- EXPECTED OUTPUT
<a href="http://google.com/" rel="nofollow">http://google.com/</a>
---- ACTUAL OUTPUT
http://google.com/
---- FAILED
