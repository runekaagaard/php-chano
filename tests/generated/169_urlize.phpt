--TEST--
A generated testfile for the "urlize" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => 'http://google.com/')));
foreach ($items as $i) echo $i->input->urlize();
--EXPECT--
<a href="http://google.com/" rel="nofollow">http://google.com/</a>