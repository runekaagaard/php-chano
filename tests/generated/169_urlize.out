Fatal error: Call to undefined method DtlIter::urlize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/169_urlize.php on line 4